lua/harpoon/mark.lua

```diff
@@ -218,6 +218,20 @@ function M.add_file(file_name_or_buf_id)
     emit_changed()
 end
 
+function M.add_current_folder()
+    local directory = vim.fn.expand("%:h")
```

Comment:
`local directory = vim.fs.dirname()`

Reply:
With now neovim 0.8 you could use also directly use `*vim.fs.dir()` to iterate over all files in the directory and add them. This would be faster and simpler.

Reply:
I wasn't aware of this! Thank you for the great suggestion. I've made a commit using the new API. It's a lot sleeker now.

Reply:
Ah, I messed up the review above. You could also replace `vim.fn.expand("%:h")` with
```lua
local directory = vim.fs.dirname(vim.api.nvim_buf_get_name(0))
```

Reply:
No worries! I made a new commit and tested that it works.

Reply:
https://github.com/ThePrimeagen/harpoon/pull/223

PS: I made this PR to fix the issue that the format action fails every time due to malformatted files from master.

Reply:
> Short and sweet. Only missing formatting I guess.

Yepp! I fixed that in another PR since it was unrelated to this one.